### DISCUSSION TITLE
Is the Ubuntu 22.04 environment ready for use in a course?

### DISCUSSION THREAD

**Discussion by jdchristensen** [sagemathinc/cocalc#6636] [Q&A] [answered]
I'm setting up a course and would prefer to use 22.04 over 20.04, but want to make sure that it's ready for use.

Thanks!

**Comment by williamstein** [comment, marked as answer]
Yes, it's definitely ready.  The main blocker was implementing https://cocalc.com/news, so we could announce it properly.

**Reply by jdchristensen** [reply]
Thanks!  I'll let you know if I find any issues.
